internal/image/docker_schema2.go

```diff
@@ -326,7 +326,7 @@ func (m *manifestSchema2) convertToManifestSchema1(ctx context.Context, options
 			ID:        v1ID,
 			Parent:    parentV1ID,
 			Comment:   historyEntry.Comment,
-			Created:   historyEntry.Created,
+			Created:   *historyEntry.Created,
```

Comment:
(This will probably go away because we just can’t change the type of `historyEntry` to the OCI struct.)

Blindly dereferencing like this can crash. Sure, the field is not optional in the format definition, but invalid/malicious configs still shouldn’t cause the code to crash.